node-api-service/server.js

```diff
@@ -247,6 +243,7 @@ async function establishTransactionSubscriptionAsync() {
     transactionSubscription.on('end', function() {
```

Comment:
@jacohend @jasonbukowski  Question: Is this necessary anymore or can this and the function `startTransactionMonitoring` be removed?

Reply:
It both handles the initialization of the lnrpc-node-client connection and starts monitoring for invoices. The latter can be dropped, but the former looks like it's still needed. Maybe stripping out the invoice subscription and renaming the method would work.

Reply:
Sorry, I misread the way the PR code was folded. `startTransactionMonitoring` has the connection creation, which we definitely still want. It's `establishTransactionSubscriptionAsync` that looks like we might not need any more of except maybe for the client settings to null?

Reply:
Ok, I've removed the transaction subscription and ran through startup and hash submission. All seems to work as expected. Going to close this as resolved, but if you have any feedback on the update, let me know.